From e12813b9d0883e02cc121ea503f4bff5fb67e9ef Mon Sep 17 00:00:00 2001
From: Mike Frysinger <vapier@gentoo.org>
Date: Fri, 8 Apr 2016 16:41:31 -0400
Subject: [PATCH xfsdump] po: respect LINGUAS build setting

It is common gettext practice to limit the translations a particular
package will include by setting the LINGUAS environment variable.

Signed-off-by: Mike Frysinger <vapier@gentoo.org>
---
 po/Makefile | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/po/Makefile b/po/Makefile
index 6d40324..48a19c1 100644
--- a/po/Makefile
+++ b/po/Makefile
@@ -6,7 +6,10 @@ TOPDIR = ..
 include $(TOPDIR)/include/builddefs
 
 POTHEAD = $(PKG_NAME).pot
-LINGUAS = de pl
+# If the user has requested a specific set of translations, only build those.
+SUPPORTED_LINGUAS = $(patsubst %.po,%,$(wildcard *.po))
+LINGUAS ?= $(SUPPORTED_LINGUAS)
+LINGUAS := $(filter $(SUPPORTED_LINGUAS),$(LINGUAS))
 LSRCFILES = $(LINGUAS:%=%.po)
 LDIRT = $(POTHEAD)
 
-- 
2.7.4

